article/sections/_1_intro_generic.tex

```diff
@@ -1,5 +1,54 @@
 \section{Introduction}
+\label{sec:intro}
 
-\blindtext[2]
+The following chapter is concerned with the existing disconnect between the 
+fields of travel demand modeling and causal inference. 
+More specifically, the chapter is motivated by the current lack of use of methods and findings from 
```

Comment:
Maybe this is a better suggestion than above. I changed "current lack of use" to "routine absence" because its a typical not total lack of use.
```suggestion
Specifically, the chapter is motivated by the routine absence of methods and findings from 
```